src/Components/Address/__tests__/useAddressAutocomplete.jest.ts

```diff
@@ -96,7 +96,7 @@ describe("useAddressAutocomplete", () => {
           })
           const { result } = setupHook({ country: "US" }, hookOptions)
 
-          await act(() => {
```

Comment:
not sure why these are suddenly failing, but in any case, this sort of invocation on act is incorrect! So better is better